src/eureka/S3_data_reduction/miri.py

```diff
@@ -436,3 +439,34 @@ def flag_bg_phot(data, meta, log):
         The updated Dataset object with outlier background pixels flagged.
     '''
     return nircam.flag_bg_phot(data, meta, log)
+
+
+def calibrated_spectra(data, meta, log):
+    """Modify data to compute calibrated spectra in units of mJy.
+
+    Parameters
+    ----------
+    data : Xarray Dataset
+        The Dataset object.
+    meta : eureka.lib.readECF.MetaClass
+        The metadata object.
+    log : logedit.Logedit
+        The current log.
+
+    Returns
+    -------
+    data : ndarray
+        The flux values in mJy
+
+    Notes
+    -----
+    History:
+
+    - 2023-07-17, KBS
+        Initial version.
+    """
+    # Convert from MJy/sr to mJy
+    log.writelog("  Calibrated MIRI spectra not yet supported...",
+                 mute=(not meta.verbose))
+
```

Comment:
I would think so, but I also haven't tested a MIRI dataset and have learned not to assume that MIRI will work the same way as the NIR instruments.

Reply:
Sounds good, no need to wait for MIRI, and I can help get MIRI integrated too since I want some calibrated spectra as well. I'd just wanted to know if you'd tried already and encountered an issue so that I knew what I'd be up against